pkg/customresource/customresource.go

```diff
@@ -114,15 +113,21 @@ func decodeSpecIntoObject(spec []byte, intoObj runtime.Object) error {
 }
 
 func createCRD(context Context, resource CustomResource) error {
-	crd, err := getCRDFromSpec(specFromResName(resource.Name))
+	c, err := rawCRDFromFile(fmt.Sprintf("%s.yaml", resource.Name))
+	if err != nil {
+		return errors.Wrap(err, "Getting raw CRD from CRD manifests")
+	}
+
+	crd, err := getCRDFromSpec(c)
 	if err != nil {
-		return err
+		return errors.Wrap(err, "Getting CRD object from CRD bytes")
 	}
+
 	ctx := contextpkg.Background()
 	_, err = context.APIExtensionClientset.ApiextensionsV1().CustomResourceDefinitions().Create(ctx, crd, metav1.CreateOptions{})
 	if err != nil {
-		if !errors.IsAlreadyExists(err) {
-			return fmt.Errorf("failed to create %s CRD. %+v", resource.Name, err)
+		if !apierrors.IsAlreadyExists(err) {
+			return fmt.Errorf("Failed to create %s CRD. %+v", resource.Name, err)
```

Comment:
```suggestion
			return errors.Errorf("Failed to create %s CRD. %+v", resource.Name, err)
```
Not your change but noticed that all the errors here are using fmt errors - perhaps create an issue to move all of them to errors pkg